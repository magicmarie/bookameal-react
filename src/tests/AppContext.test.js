import React from "react";
import { shallow } from "enzyme";
import AppProvider from "../appContext";

describe("app context api", () => {
  let wrapper;
  beforeEach(() => {
    wrapper = shallow(<AppProvider />);
  });
  it("should render correctly", () => {
    expect(wrapper).toMatchSnapshot();
  });
});
