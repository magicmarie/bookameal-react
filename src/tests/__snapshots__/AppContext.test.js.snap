// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`app context api should render correctly 1`] = `
ShallowWrapper {
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <AppProvider />,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "function",
    "props": Object {
      "children": undefined,
      "value": Object {
        "cartItems": Array [],
        "clearCart": [Function],
        "deleteFromCart": [Function],
        "email": "test@test.com",
        "getCart": [Function],
        "isAdmin": true,
        "isLoggedIn": true,
        "isUser": false,
        "login": [Function],
        "logout": [Function],
        "setCart": [Function],
        "setQuantity": [Function],
      },
    },
    "ref": null,
    "rendered": null,
    "type": Object {
      "$$typeof": Symbol(react.provider),
      "_context": Object {
        "$$typeof": Symbol(react.context),
        "Consumer": [Circular],
        "Provider": [Circular],
        "_calculateChangedBits": null,
        "_changedBits": 0,
        "_changedBits2": 0,
        "_currentRenderer": null,
        "_currentRenderer2": null,
        "_currentValue": undefined,
        "_currentValue2": undefined,
        "_defaultValue": undefined,
      },
    },
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "function",
      "props": Object {
        "children": undefined,
        "value": Object {
          "cartItems": Array [],
          "clearCart": [Function],
          "deleteFromCart": [Function],
          "email": "test@test.com",
          "getCart": [Function],
          "isAdmin": true,
          "isLoggedIn": true,
          "isUser": false,
          "login": [Function],
          "logout": [Function],
          "setCart": [Function],
          "setQuantity": [Function],
        },
      },
      "ref": null,
      "rendered": null,
      "type": Object {
        "$$typeof": Symbol(react.provider),
        "_context": Object {
          "$$typeof": Symbol(react.context),
          "Consumer": [Circular],
          "Provider": [Circular],
          "_calculateChangedBits": null,
          "_changedBits": 0,
          "_changedBits2": 0,
          "_currentRenderer": null,
          "_currentRenderer2": null,
          "_currentValue": undefined,
          "_currentValue2": undefined,
          "_defaultValue": undefined,
        },
      },
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
        "lifecycles": Object {
          "componentDidUpdate": Object {
            "onSetState": true,
          },
          "getSnapshotBeforeUpdate": true,
        },
      },
    },
    "attachTo": undefined,
    "hydrateIn": undefined,
  },
}
`;
